webhooks/views.py

```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from interface.models import Repo
+
+from webhooks.models import GitHook, GitHookError, GitHookAccessError
+
+
+@csrf_exempt
+def github_webhook(request):
+    git_hook = GitHook(request)
+
+    try:
+        git_hook.process_request()
+    except GitHookAccessError:
+        return HttpResponse(status=403)
+    except GitHookError as e:
+        return HttpResponse(str(e), status=400)
+
+    repo = Repo.objects.filter(full_name=git_hook.repository['full_name']).first()
```

Comment:
Using exceptions for default behavior is a generally bad practice. Using exceptions should remain an exception, IMO. Moreover, mine is only one line 💪